<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My school</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(250, 218, 221)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 236, 236);
font: 63 15pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QPushButton" name="AddLangButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(250, 218, 221);
font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
      </property>
      <property name="text">
       <string>Add Lang</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <widget class="QPushButton" name="AddGroupsButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(250, 218, 221);
font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
      </property>
      <property name="text">
       <string>Add Groups</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" rowspan="2">
     <widget class="QPushButton" name="EditButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(250, 218, 221);
font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
      </property>
      <property name="text">
       <string>&amp;Edit</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="AddStudentButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(250, 218, 221);
font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
      </property>
      <property name="text">
       <string>&amp;Add student</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
